testing/trino-testing/src/main/java/io/trino/testing/BaseFailureRecoveryTest.java

```diff
@@ -148,49 +131,6 @@ public void initTables()
 
     protected abstract boolean areWriteRetriesSupported();
 
-    @Test(invocationCount = INVOCATION_COUNT)
```

Comment:
Where are tests removed from this class now. 
We may change code so those are not run for all subclasses. But we cannot just drop those (at least without clear explanation why we are dropping them)

Reply:
(this applies to all lines removed from this class)

Reply:
IIRC the plan was to have a "full-coverage" test run on Hive. And slimmer class to serve as a base for connector-specific test classes.

Reply:
Not sure how i forgot to do this initially. I took a slightly different approach than we had discussed - instead, adding an `ExtendedFailureRecoveryTest`, which only the BaseHiveFailureRecoveryTest extends.

Reply:
Good job! Thanks for cleaning this up